doc/sphinx/addendum/type-classes.rst

```diff
@@ -407,8 +407,13 @@ Summary of the commands
 
    Shows the list of instances associated with the typeclass :token:`reference`.
 
+.. cmd:: Print Typeclasses
```

Comment:
I'd put the `:cmd:`s on adjacent lines with a `:name: Print Typeclasses; Print TypeClasses` on the next line, then the deprecation should say which form is deprecated.

Reply:
I feel like putting them on the same line is confusing. Perhaps I can add documentation to `Print TypeClasses` pointing to the correct form above.

Reply:
They would be on separate lines, like this:

![image](https://user-images.githubusercontent.com/1253341/196288347-9893dd87-e303-4d71-b5f2-b8b592f02692.png)

Do whichever you prefer.